CIP-xxxx/README.md

```diff
@@ -0,0 +1,111 @@
+---
+CPS: ????
+Title: SRV registry
+Status: Active
+Category: Network
+Authors:
+  - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+Implementors: N/A
+Discussions: 
+  - [PR](https://github.com/cardano-foundation/CIPs/pull/xxx)
+Created: 2025-04-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines the procedure for assigning SRV prefixes for Cardano applications (like `cardano-node`, `mithril`, etc.).
+
+## Motivation
+
+The **Cardano Ledger** allows the use of SRV records in the **SPO** registration certificate.
+
+In this CIP, we propose to make them usable by decentralised applications running on `Cardano`.
+
+Having access to the ledger (either directly or through tools like `cardano-cli`) will allow decentralised applications (DAPS) developers to construct networks based on registered relays. 
+
+All involved parties (SPOs, DAP developers, etc) need to agree on SRV prefixes used by various DAPs and thus a public registry governed by a public CIP process is required to foster DAPs that co-exist with the **Cardano** network, like **Mithril** (see [CIP#137]).
```

Comment:
```suggestion
All involved parties (SPOs, DAP developers, etc) need to agree on SRV prefixes used by various DAPs and thus a public registry governed by a public CIP process is required to foster DAPs that co-exist with the **Cardano** network, like **Mithril** (see [CIP#137](../CIP-0137).
```

Reply:
The link is included at the bottom, I'll update it to a relative one, though.